scripts/hotrod_dummy_test.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
we already have a script that does basic hotrod test: `scripts/hotrod-integration-test.sh`. The objective is to make the checks stronger, not to duplicate.